lib/matplotlib/axes/_axes.py

```diff
@@ -2475,7 +2475,7 @@ def pie(self, x, explode=None, labels=None, colors=None,
             autopct=None, pctdistance=0.6, shadow=False, labeldistance=1.1,
             startangle=None, radius=None, counterclock=True,
             wedgeprops=None, textprops=None, center=(0, 0),
-            frame=False):
+            frame=False, rotatelabels=False):
         r"""
```

Comment:
Do you mind converting this docstring into numpydoc? It will make it much more readable.